test/run_pass/test_process_stats.js

```diff
@@ -0,0 +1,15 @@
+var bufs = []
+var lastUsage = process.memoryUsage()
+setInterval(() => {
+  var buf = Buffer.allocUnsafe(1024)
+  buf.fill('rokid')
+  bufs.push(buf)
+  var usage = process.memoryUsage()
+  console.log('rss:' + usage.rss +
+    ' total:' + usage.heapTotal + 
+    ' used:' + usage.heapUsed + 
+    ' used delta:' + (usage.heapUsed - lastUsage.heapUsed) +
+    ' bufLen:' + bufs.length
+  )
```

Comment:
We need some asserts as well :)

Reply:
this file is a case for this pr, i will remove it later